openssl/src/symm.rs

```diff
@@ -12,6 +36,10 @@ pub enum Mode {
     Decrypt,
 }
 
+/// Represent a particular cipher algorithm.  See OpenSSL doc at [`EVP_EncryptInit`] for more
```

Comment:
The first paragraph of documentation should be a short single sentence since it is shown in the parent page of the documentation. Can you move the other sentences here and below down to their own paragraph?

Reply:
Please check if it is better now.